packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationGeneralPage.tsx

```diff
@@ -98,7 +99,10 @@ const OrganizationProfileSection = () => {
           fallback={profile}
         >
           <Action.Closed value='edit'>
-            <ProfileSection.Item id='organizationProfile'>
+            <ProfileSection.Item
+              id='organizationProfile'
+              sx={t => ({ maxHeight: t.space.$8 })}
```

Comment:
Why is this needed?

Reply:
This is mainly to avoid having `Organization Profile` text centered, as it solves the layout shift when the action card is open, and also it's a bit closer to the design.